Nodejs/Product/Nodejs/Repl/NpmReplCommand.cs

```diff
@@ -76,19 +76,9 @@ class NpmReplCommand : IReplCommand {
                     continue;
                 }
 
-                EnvDTE.Properties properties = dteProject.Properties;
-                if (dteProject.Properties == null) {
-                    continue;
-                }
-
                 string projectName = dteProject.Name;
-                EnvDTE.Property projectHome = properties.Item("ProjectHome");
-                if (projectHome == null || projectName == null) {
-                    continue;
-                }
-
-                var projectDirectory = projectHome.Value as string;
-                if (projectDirectory != null) {
+                string projectDirectory = Path.GetDirectoryName(dteProject.FullName);
```

Comment:
Surely no one would do such an unthinkable thing ;)

Any thoughts on what property to use on the `dteProject` or the `projectHome` to get this then? Do you recommend restoring the previous `ProjectHome` logic for use on node projects, and only using this fallback logic for other project types?